SingularityRunnerBase/src/main/java/com/hubspot/singularity/runner/base/shared/S3UploadMetadata.java

```diff
@@ -79,7 +80,8 @@ public S3UploadMetadata(@JsonProperty("directory") String directory,
                           @JsonProperty("uploaderType") Optional<SingularityUploaderType> uploaderType,
                           @JsonProperty("gcsCredentials") Map<String, Object> gcsCredentials,
                           @JsonProperty("gcsStorageClass") Optional<String> gcsStorageClass,
-                          @JsonProperty("encryptionKey") Optional<String> encryptionKey) {
+                          @JsonProperty("encryptionKey") Optional<String> encryptionKey,
+                          @JsonProperty("s3ServerSideEncryption") Optional<Boolean> s3ServerSideEncryption) {
```

Comment:
Can we name this something more like `useS3ServerSideEncryption`? Makes it a bit more obvious it's a boolean and not a string to specify a type of encryption. Other than that PR looks good 👍